.travis.yml

```diff
@@ -1,12 +1,29 @@
 language: scala
-scala:
-  - 2.10.4
-jdk:
-  - openjdk7
-  - openjdk6
 sudo: false
+cache:
+  directories:
+    - $HOME/.ivy2
```

Comment:
By the way, if you have any other Java/Scala libraries that you test on Travis then this cache option is really nice; should significantly speed up your builds.